config/capsule.migrations/20161121212031_move_capsule_to_foreman_proxy_content.rb

```diff
@@ -0,0 +1,9 @@
+# migrate answers from capsule if exist
+answers['capsule'].each do |k|
+  if k.eql? 'capsule'
+    answers['foreman_proxy_content']['foreman_proxy_content'] = answers['capsule'].delete k
```

Comment:
What's this supposed to be doing? A user will have an answers YAML like this:

```yaml
---
capsule:
  qpid_router_logging_path: 'whatever'
```

We just need that to become:

```yaml
---
foreman_proxy_content:
    qpid_router_logging_path: 'whatever'
```

So I think you can just do this or something similar:

```ruby
answers['foreman_proxy_content'] = answers.delete('capsule') if answers.key?('capsule')
```

None of the option names in the class itself have the word capsule in them:
https://github.com/Katello/puppet-capsule/blob/master/manifests/init.pp#L65-L91